external/webserver/http_client.c

```diff
@@ -237,7 +264,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 					}
 					if (strcmp(key, "Content-Length") == 0) {
 						len->content_len = HTTP_ATOI(value);
-						response->total_len = len->content_len;
+						if(response){
```

Comment:
Please leave spaces as per the coding rules. "if(response){" --> "if (response) {"

Reply:
done